### PR TITLE
Feature/vta 695

### PR DESCRIPTION
**Pull request by IuliaDVSA**

### CODE REVIEW

**Review by blondeholly — CHANGES_REQUESTED** [review]

File: src/main/java/clients/TestStationsClient.java [hunk 1]
```diff
@@ -41,4 +51,15 @@ private Response callGetTestStations(Filter filter) {
 
         return response;
     }
+
+    private Response callGetTestStationsEmail(String p_umber) {
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/main/java/clients/TestStationsClient.java:55]: Yep, :) changed

File: src/test/java/atf/GetTestStations.java [hunk 2]
```diff
@@ -22,17 +24,32 @@ public void testStationReferenceData() {
...
-    @Title("CVSB-7949 - AC1 - API Consumer retrieve test stations reference data - testStationStatus : active")
+    @Title("CVSB-7949 - AC1 - API Consumer retrieve test stations reference data - testStationStatus : active or termination requested")
     @Test
     public void testStationReferenceDataStationStatus() {
         testStationSteps.getTestStationsWithData();
         testStationSteps.statusCodeShouldBe(200);
         testStationSteps.validateData(TestStationsData.buildTestStationData());
         testStationSteps.validateEveryRecordHasField("testStationStatus");
-        testStationSteps.validateEveryRecordHasFieldValue("testStationStatus", "active");
+        testStationSteps.validateEveryRecordHasFieldValue("testStationStatus", TestStationsData.buildTestStationData());
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/test/java/atf/GetTestStations.java]: Removed, this should not have been here.

File: src/test/java/testresults/GetTestResults.java [hunk 3]
```diff
@@ -18,14 +18,13 @@
...
 import util.JsonPathAlteration;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.UUID;
-
 import static util.DataUtil.generateRandomExcludingValues;
 
+@WithTag("In_test")
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/test/java/testresults/GetTestResults.java]: Removed

File: src/test/java/atf/GetTestStations.java [hunk 4]
```diff
@@ -22,17 +24,32 @@ public void testStationReferenceData() {
         testStationSteps.getTestStationsWithData();
         testStationSteps.statusCodeShouldBe(200);
         testStationSteps.validateData(TestStationsData.buildTestStationData());
-
     }
 
-    @Title("CVSB-7949 - AC1 - API Consumer retrieve test stations reference data - testStationStatus : active")
+    @Title("CVSB-7949 - AC1 - API Consumer retrieve test stations reference data - testStationStatus : active or termination requested")
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/test/java/atf/GetTestStations.java]: Removed

File: src/main/java/clients/TestStationsClient.java [hunk 5]
```diff
@@ -41,4 +51,15 @@ private Response callGetTestStations(Filter filter) {
...
+        Response response = given().filters(new BasePathFilter())
+//                .filters(new BasePathFilter())
+                .contentType(ContentType.JSON)
+                .pathParam("p_umber", p_umber)
+                .log().method().log().uri().log().body()
+                .get("/test-stations/{p_umber}");
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/main/java/clients/TestStationsClient.java]: Changed

File: src/main/java/clients/TestStationsClient.java [hunk 6]
```diff
@@ -41,4 +51,15 @@ private Response callGetTestStations(Filter filter) {
 
         return response;
     }
+
+    private Response callGetTestStationsEmail(String p_umber) {
+        Response response = given().filters(new BasePathFilter())
+//                .filters(new BasePathFilter())
+                .contentType(ContentType.JSON)
+                .pathParam("p_umber", p_umber)
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/main/java/clients/TestStationsClient.java]: Changed

File: src/main/java/clients/TestStationsClient.java [hunk 7]
```diff
@@ -41,4 +51,15 @@ private Response callGetTestStations(Filter filter) {
 
         return response;
     }
+
+    private Response callGetTestStationsEmail(String p_umber) {
+        Response response = given().filters(new BasePathFilter())
+//                .filters(new BasePathFilter())
```
[the remark this answers is not in the document]
Comment (IuliaDVSA) [reply src/main/java/clients/TestStationsClient.java]: Changed

**Comment by ZeypEr** [comment]
VTA-695 has been approved and reviewed.

**Review by ZeypEr — APPROVED** [review]
VTA-695 has been approved and reviewed.

**Review by blondeholly — APPROVED** [review]

**Comment by IuliaDVSA** [comment]
VTA-695 reviewed and merged

**Pull request merged**